webgui/scripts/codecompass/view/fileManager.js

```diff
@@ -237,6 +239,46 @@ function (on, query, style, topic, declare, Memory, Observable,
       path.unshift('project');
 
       this.set('path', path);
+      this.set('filePath', fileInfo.path);
+    },
+
+    _setFilePathAttr : function (path) {
```

Comment:
We don't need a public interface for setting the file path in the header. The invariant should be that always the current file is indicated there, and we should't let the user setting it from the outside.